sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -27,7 +27,7 @@
 }
 
 .secondary-color-light {
-  color: $secondary-header-color-lt;
+  color: $secondary-header-color-lt !important;
```

Comment:
do we really need `!important`?

Reply:
Yes, the font color does not reflect on the formio side without the !important property